### PR TITLE
[Fixes: #4929] Remove Google+ from share options, it does not exist anymore

### PR DESCRIPTION
**Pull request by LiliGuimaraes**
<Include a few sentences describing the overall goals for this Pull Request>

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

For all pull requests:

- [x] Confirm you have read the [contribution guidelines](https://github.com/GeoNode/geonode/blob/master/CONTRIBUTING.md) 
- [x] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in documentation)
- [ x] Make sure the first PR targets the master branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for contrib modules):
- [x] There is a ticket in https://github.com/GeoNode/geonode/issues describing the issue/improvement/feature (a notable exemptions is, changes not visible to end users)
- [x] The issue connected to the PR must have Labels and Milestone assigned
- [x] PR for bug fixes and small new features are presented as a single commit
- [x] Commit message must be in the form "[Fixes #<issue_number>] Title of the Issue"
- [x] New unit tests have been added covering the changes, unless there are explanation on why the tests are not necessary/implemented
- [x] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [x] This PR passes the QA checks: flake8 geonode
- [x] Commits changing the **settings**, **UI**, **existing user workflows**, or adding **new functionality**, need to include documentation updates

**Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.**

### CODE REVIEW

**Comment by t-book** [comment]
Hi @LiliGuimaraes,

thanks a lot for your contribution. From a quick look, I think your PR misses the removal of plusone.js which as far as I remember is for google+ sharing: https://github.com/GeoNode/geonode/search?q=plusone&unscoped_q=plusone

Further, as OSGEO project we need a signed CLA from every contributor, find more about it here: https://github.com/GeoNode/geonode/blob/master/CONTRIBUTING.md

**Comment by LiliGuimaraes** [comment]
@t-book and @afabiani 
Just sent email with signed Contribution License Code for entry into OsGEO contributor group

File: geonode/documents/templates/documents/document_detail.html [hunk 1]
```diff
@@ -251,16 +251,6 @@ <h4>{% trans "Permissions" %}</h4>
...
-    <script type="text/javascript">
-    {% autoescape off %}
-        (function() {
-            var po = document.createElement('script'); po.type = 'text/javascript'; po.async = true;
-            po.src = 'https://apis.google.com/js/plusone.js';
-            var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(po, s);
-        })();
-    {% endautoescape %}
-    </script>
     {% endif %}
```
Comment (t-book) [inline geonode/documents/templates/documents/document_detail.html]: This {% endif %} is obsolete and has to be removed.

File: geonode/layers/templates/layers/layer_detail.html [hunk 2]
```diff
@@ -747,15 +747,6 @@ <h1>{% trans "Processing..." %}</h1>
...
-    {% if DISPLAY_SOCIAL %}
-    {% autoescape off %}
-     <script type='text/javascript'>
-     (function() {
-       var po = document.createElement('script'); po.type = 'text/javascript'; po.async = true;
-       po.src = 'https://apis.google.com/js/plusone.js';
-       var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(po, s);
-     })();
-     {% endautoescape %}
     {% endif %}
```
Comment (t-book) [inline geonode/layers/templates/layers/layer_detail.html]: This {% endif %} is obsolete and has also to be removed.

**Comment by afabiani** [comment]
> @t-book and @afabiani
> Just sent email with signed Contribution License Code for entry into OsGEO contributor group

Thanks @LiliGuimaraes adding you to the contributors'list.

In order to finalize the PR we need to:

1. Make sure there's a GitHub issue related to this PR
2. Change the title of the PR accordingly "[Fixes #<issue_num>] ... "
3. Make sure all the Travis tests pass

Thanks again.
Alessio.

**Comment by t-book** [comment]
@LiliGuimaraes I've just changed the title. With my requested changes Travis should finish. Thanks again and welcome to the community ;)

**Comment by t-book** [comment]
Travis is failing because: 

> "NameError: name 'homepage' is not defined"

It looks home has been (accidentally?) removed from urls.py
https://github.com/GeoNode/geonode/pull/4995/files#diff-53064682d0891fb2cfb01239348945e9L61

**Comment by LiliGuimaraes** [comment]
I've already open a new PR to check this issue
Please, finish this one and considering the other.
Thanks

**Pull request closed without merging**